src/splat/segtypes/common/segment.py

```diff
@@ -1,4 +1,4 @@
-from ...segtypes.segment import Segment
+from ...segtypes.segment import Segment, SegmentType
```

Comment:
Is there a reason for this change?

Reply:
Just a type alias to make the types for the code I introduced more self-documenting &mdash; certainly not necessary

Reply:
ahh I hadn't understood the full context of it. thank you!